AutomatedTesting/Gem/PythonTests/Prefab/tests/spawnables/Lua_Spawnables_SimpleSpawnAndDespawn.py

```diff
@@ -51,7 +51,7 @@ def validate_entities_in_edit_mode(condition):
                                       expected_spawned_entity_scale)
 
     # Wait for Lua script to exit Game Mode on despawn of PinkFlower.spawnable and search for expected entities
-    game_mode_exited = helper.wait_for_condition(lambda: not general.is_in_game_mode(), 5.0)
+    game_mode_exited = helper.wait_for_condition(lambda: not general.is_in_game_mode(), 10.0)
```

Comment:
These increased times look troubling. Does exiting game mode with a simple spawnable really take 10 seconds? 

I know that a lot of trial and error had to be done to enable these tests. Just want to make sure that debug info is not being committed here